microsoft-edge/devtools-guide-chromium/javascript/breakpoints.md

```diff
@@ -179,50 +182,60 @@ Use exception breakpoints when you want to pause on the line of code that is thr
 
 1. Select the **Sources** tool.
 
-1. Click **Pause on exceptions** (![Pause on exceptions](./breakpoints-images/pause-on-exceptions-icon.png)).  The icon turns blue when enabled.
-
-   ![The Pause on exceptions button](./breakpoints-images/javascript-sources-page-js-pause-on-exceptions.png)
+1. In the **Breakpoints** panel, select **Pause on uncaught exceptions** to pause when the code throws a JavaScript exception.
 
-1. **Optional:** Check the **Pause On Caught Exceptions** checkbox if you also want to pause on caught exceptions, in addition to uncaught ones.
+1. **Optional:** Select **Pause on caught exceptions** if you also want to pause on caught exceptions, for example when an exception is caught by a `try/catch` block.
```

Comment:
```suggestion
1. **Optional:** Select **Pause on caught exceptions** if you also want to pause on caught exceptions - for example, when an exception is caught by a `try/catch` block:
```

Reply:
rewrote the two items as:

1. To pause when the code throws a JavaScript exception, in the **Breakpoints** pane, select the **Pause on uncaught exceptions** checkbox.

1. To pause on caught exceptions, such as when an exception is caught by a `try/catch` block, select the **Pause on caught exceptions** checkbox: